internal/luhn.go

```diff
@@ -0,0 +1,38 @@
+package internal
+
+import (
+	"regexp"
+)
+
+var nonNumberRegExp = regexp.MustCompile(`[^\d]+`)
+
+// ValidateLuhn checks if a string of digits passes the Luhn algorithm validation.
+//
+// `number` must contain only numeric characters.
+//
+// [Source]
+//
+// [Source]: https://github.com/luhnmod10/go
+func ValidateLuhn(number string) bool {
```

Comment:
The `/regimes/common/luhn.go` already implements this.

Reply:
I'll move the Luhn tests there since I do see value in having those

Reply:
And as an extra: the Luhn algorithm being something so general (it can be used outside of regimes, such as for validating credit card numbers) should it still reside in `regimes/common` or is `internal` a better place?